decksite/views/cards.py

```diff
@@ -9,11 +9,13 @@
 
 # pylint: disable=no-self-use
 class Cards(View):
-    def __init__(self, cards: List[Card], tournament_only: bool = False) -> None:
+    def __init__(self, cards: List[Card], tournament_only: bool = False, query: str = '') -> None:
         super().__init__()
         self.show_seasons = True
         self.show_tournament_toggle = True
         self.tournament_only = tournament_only
+        self.query = query
+        self.cards = cards
```

Comment:
It’s surprising to me that this view wasn’t setting self.cards already. Without that how were the cards getting vivified for display? How were they being shown in the mustache. Not sure what problems this might cause but it made my spidey sense tingle.

Reply:
This line was unnecessary, self.cards was being set a few lines lower already.